packages/rich-text/src/component/index.native.js

```diff
@@ -59,6 +59,8 @@ const gutenbergFormatNamesToAztec = {
 	'core/strikethrough': 'strikethrough',
 };
 
+const EMPTY_MULTILINE_PARAGRAPH = '<p></p>';
```

Comment:
Not a big deal, but what is the significance of "MULTILINE" in this name? Is there a non-multiline paragraph we're distinguishing?

Reply:
Good point, updated to better name here: https://github.com/WordPress/gutenberg/pull/22561/commits/8f9bd436b901e8cd3cadebd8fe36b574bd23c0b9